server/src/pages/shot/model.js

```diff
@@ -27,11 +27,11 @@ exports.createModel = function(req) {
     id: req.shot.id,
     productName: req.config.productName,
     isExtInstalled: !!req.deviceId,
-    isOwner: req.deviceId === req.shot.ownerId || isFxaAuthenticated,
+    isOwner: req.shot.isOwner(req.deviceId, req.accountId),
     isFxaAuthenticated,
     gaId: req.config.gaId,
     deviceId: req.deviceId,
-    authenticated: !!req.deviceId,
+    authenticated: !!(req.deviceId || req.accountId),
```

Comment:
I'm unclear why this is set here, and also in reactrender. Probably for no good reason ;)